refresh_test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2017 ScyllaDB
+
+import time
+
+from avocado import main
+from sdcm.tester import ClusterTester
+from sdcm.nemesis import RefreshAllNodesMonkey
+
+
+class RefreshTest(ClusterTester):
+    """
+    Nodetool refresh after uploading lot of data to a cluster with running load in the background.
+
+    1) start cassandra-stress in background to prepare data
+    2) when stress workload finishes 2/3, start refresh nemesis
+    3) stop refresh when stress workload is about finished
+
```

Comment:
What does "nodetool refresh" do when you don't load new sstables to the system?
Don't you need to copy offline sstables to the system before refreshing?
And why do we need to stop the refresh?